opus/application/static_media/js/browse.js

```diff
@@ -327,7 +327,7 @@ var o_browse = {
             }
 
             // remove the sort pill right away
-            $(this).closest("li").remove();
+            $(this).closest(".list-inline-item").remove();
```

Comment:
so... what I am trying desperately to avoid is using "closest", as that could change w/future things.  Hence my suggestion to use something a little more opus-definitive that will get you exactly what you want w/out having to use "closest".  Sorry, i should have been clearer in my original comment.